app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', counter_cache: :posts_counter
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.last(5)
+  end
```

Comment:
- [ ] Kindly make sure your `Post` model has a method that updates the `posts counter` for a user as required for the project.
```ruby
...
def update_post_counter
  # Your code here....
end
```

- [ **OPTIONAL** ] **IMPORTANT:** When implementing the `update_post_counter` method, I suggest you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to the database.

Reply:
Changes made